pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -87,7 +87,7 @@ def default_pypeit_par(cls):
         par['reduce']['findobj']['find_trim_edge'] = [2,2]    # Slit is too short to trim 5,5 especially
         par['reduce']['skysub']['bspline_spacing'] = 0.8
         par['reduce']['skysub']['global_sky_std']  = False    # Do not perform global sky subtraction for standard stars
-        par['reduce']['skysub']['no_poly'] = True             # Do not use polynomial degree of freedom for global skysub
+        par['reduce']['skysub']['no_poly_global'] = True      # Do not use polynomial degree of freedom for global skysub
```

Comment:
Hmm, I wonder if this is why we have sky-subtraction problems for GNIRS? I'm confused why we don't do this for other fixed format echelle's but maybe because the GNIRS slit is narrower. Maybe add a note here asking if this is the right behavior as a TODO, but I would not change with investigation.